### DISCUSSION TITLE
Issues in Serene 8.4.7.1 new project

### DISCUSSION THREAD

**Discussion by vinnaykulkarni** [serenity-is/Serenity#7132] [Help] [answered]
1. The image upload editor shows an error while uploading images, which happens even with the default user module as well - Error: "Uploaded file is not image" even if the file extension is right.
2. When we click on the category links the dialog exits

**Comment by Surenthiran** [comment]
i also had the same problem while upgrading from 8.1 to the latest manually. unable to figure out the issue.

**Comment by volkanceylan** [comment, marked as answer]
Change return type of HandleUploadRequest in FilePage to UploadResponse instead of ServiceResponse.

Category link issue is also in demo / StartSharp but as they are hidden it was not noticed. It will be fixed in next version, but category links are deprecated and will be removed completely. You may add following to site.css for now:

.category-links {
  display: none;
}

**Reply by vinnaykulkarni** [reply]
Thanks changing the return type have resolved the issue
